gauge-display/script.js

```diff
@@ -202,44 +202,41 @@ function switchNight(event) {
     let readoutDisplayElements = document.querySelectorAll('.number-display');
     let gaugePageBackground = document.getElementById('gauges');
 
+    const addClassToElement = (element, classToAdd) =>  element.classList.add(classToAdd);
+    const removeClassFromElement =  (element, classToRemove) =>  element.classList.remove(classToRemove);
+
+    const nightModeApplicableElements = [
+        nightModeButton,
+        ledUpButton,
+        ledDownButton,
+        chartBackground,
+        gaugePageBackground,
+        ...readoutLabelElements,
+        ...readoutElements,
+        ...readoutDisplayElements,
+    ]
+
     if (night) { //switching to day
-        nightModeButton.classList.remove('icono-sun');
-        nightModeButton.classList.remove('night');
-        nightModeButton.classList.add('icono-moon');
-        ledUpButton.classList.remove('night');
-        ledDownButton.classList.remove('night');
-        chartBackground.classList.remove('night');
-        gaugePageBackground.classList.remove('night');
-        readoutLabelElements.forEach(readoutLabel => readoutLabel.classList.remove('night'));
-        readoutElements.forEach(readoutElement => readoutElement.classList.remove('night'));
-        readoutDisplayElements.forEach(displayElement => displayElement.classList.remove('night'));
+        nightModeApplicableElements.forEach((element) => removeClassFromElement(element, 'night'))
```

Comment:
Rather than adding classes to a bunch of elements you could leverage a "dark mode" switch and the media query for that. Works really well when combines with custom properties for things like background color and font color.
https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

Reply:
Also, it was amateur hour on adding the classes to a billion things, versus just adding it to one parent